Gemfile

```diff
@@ -11,13 +11,18 @@ group :backend, :frontend, :core, :api do
   else
     gem 'rails', ENV['RAILS_VERSION'] || '~> 6.1.0', require: false
   end
+
+  if ENV['RAILS_VERSION'] == '7.0.0'
```

Comment:
Does this need to be adjusted to compare against any version later than 7.0.0 since 7.0.1 is already out?

Thank you for opening this PR, by the way, we'd love to bump to Rails 7 ASAP and Solidus is one of the few things preventing that for us.

Reply:
Yes, this needs to be a bit smarter. Good catch!